lib/credentials.js

```diff
@@ -146,5 +146,23 @@ AWS.Credentials = AWS.util.inherit({
   refresh: function refresh(callback) {
     this.expired = false;
     callback();
+  },
+
+  /**
+   * @api private
+   */
+  addPromisesToClass: function addPromises(PromiseDependency) {
```

Comment:
Yup, I meant to change all of the function names to match their fields, but I missed that one.